addons/aircraft/XEH_PREP.hpp

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
One of the most difficult thing for me in this PR was creating new ACE-specific files without templates but with many samples in the source tree. And unfortunately these samples are different in some aspects like these spaces at file start.